flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1191,6 +1191,12 @@
     "com.ulduzsoft.Birdtray": {
         "finish-args-flatpak-spawn-access": "the app predates this linter rule"
     },
+    "com.ulaa.Ulaa": {
+        "flathub-json-modified-publish-delay": "extra-data",
```

Comment:
this should be dropped https://github.com/flathub/flathub/pull/6050/files#r1922556481